client/src/frontend/components/resource-slide/Slide/index.js

```diff
@@ -0,0 +1,55 @@
+export * from "./Info";
+export Zoom from "./Zoom";
```

Comment:
These exports here are weird. Why are you doing this?

Reply:
It was so I could access these default slide components from other slides, such as the Interactive and Image slides which use the zoom button, however it is more important that I keep the code consistent. I moved the files so the exports could be more consistent with the other slide components.